### DISCUSSION TITLE
Mercadolivre disponibiliza Forum e pede que o utilizem

### DISCUSSION THREAD

**Discussion by ernaninfilho** [rodurma/forum-meli#24] [Geral]
Olá pessoal. O Mercadolivre voltou a disponibilizar seu forum, e pedem que o utilizem. Estão monitorando e coletando feedbacks, para melhoria da API.

Então, a pedido do próprio Mercadolivre, estou deixando aqui o link do forum:

https://developersforum.mercadolivre.com.br/pt/web/forum-op/

Há ainda alguns bugs, mas creio que logo resolvam. O importante é utilizarem, para que eles vejam quais são as necessidades da API (alô produtos da categoria fashion). Então, participem.

Sucesso!

**Comment by DuPiratello** [comment]
Estou tentado acessar mas sem sucesso. recebo o erro:
<img width="1404" height="824" alt="imagem_2025-10-21_132537453" src="https://github.com/user-attachments/assets/4242bdae-27a1-485d-8865-569250589c73" />
